repairnator/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/step/paths/TestComputePlugins.java

```diff
@@ -76,7 +76,7 @@ public void testComputePluginsWithSingleModuleProject() throws IOException {
             assertThat(stepStatus.isSuccess(), is(true));
         }
 
-        assertTrue(jobStatus.getProperties().getProjectMetrics().getNumberPlugins() >= 7);
```

Comment:
here you undo a change we made to fix flakiness, probably a missing merge against master?

Reply:
sorry about that, I kind of missed this due to changes in privileges that we now have to commit into fork instead.